src/data/valid/Database-extraction_set-exhaustive.yaml

```diff
@@ -7,7 +7,7 @@ extraction_set:
     description: DNA extraction of NEON sample WREF_072-O-20190618-COMP using SOP BMI_dnaExtractionSOP_v7
     #enforce analytical sample typecode?
     has_input:
-      - generic:xxx
+      - nmdc:procsm-99-123456
     #cardinality of outputs should be 1-many, may want to fractionate samples or keep supernatant
     has_output:
       - generic:xxx
```

Comment:
If we've done things correctly generic:xxx should not be valid for has_output for this class. It should only allow an instance of Class ProcessedSample.